src/Components/ProvisioningWizard/steps/FinishProgress/index.js

```diff
@@ -128,11 +138,14 @@ const FinishStep = ({ imageID, setLaunchSuccess }) => {
               <br />
               {polledReservation?.status}
               <span className="status-error">
+                {isPollingTimeout && 'Session timeout'}
+                <br />
                 {awsReservationError?.response?.data?.msg}
+                <br />
                 {pubkeyError?.response?.data?.msg}
                 {polledReservation?.error}
               </span>
-              {reservationID && <input type="hidden" name="reservation_id" value={reservationID} />}
+              {reservationID && <input readOnly name="reservation_id" value={reservationID} />}
```

Comment:
I added a readOnly because of a console error, if there is a value with no `onChange` it requires a `readOnly` flag, we can fix it in another PR though.

The `hidden` was removed accidentally